### PR TITLE
Mutt2khal: pass more robust date formats to khal and shell escape characters

### PR DESCRIPTION
**Pull request by Konfekt**
pass more robust date formats to khal and shell escape characters

see also https://github.com/pimutils/khal/issues/1078

### CODE REVIEW

File: misc/mutt2khal [hunk 1]
```diff
@@ -37,7 +40,7 @@
 }
 
 END {
-  print "khal new", loc, d_st, t_st, d_end, t_end, summ, desc | "sh"
+  printf("khal new --location %s %sT%s %sT%s %s :: %s", loc, d_st, t_st, d_end, t_end, summ, desc) | "sh"
```
Comment (d7415) [inline misc/mutt2khal:43]: The `T`s here will break the default configuration.

Comment (Konfekt) [reply misc/mutt2khal:43]: Yes, but according to https://github.com/pimutils/khal/issues/1078#issuecomment-896695860 it is the only way to currently pass dates to khal.

Comment (Konfekt) [reply misc/mutt2khal]: Okay, got it, khal only accepts the one date and time format specified in the config; I somehow thought it always tries to interpret all formats thrown at it (independent of the configuration).

Comment (Konfekt) [reply misc/mutt2khal]: Corrected it  to assume the default format again.

**Comment by geier** [comment]
closing and opening to force tests to re-run

**Pull request merged**